chore(repo): update maintenance status (#392)
ionic-team/stencil-ds-output-targets: .github/PULL_REQUEST_TEMPLATE.md

## Pull request checklist

<!-- Please note that this repository is largely maintained for the purposes of supporting the Ionic Framework -->

> <!-- As a result, we may not immediately get around to reviewing pull requests that do not support the Framework (or at all) -->

Reviewer: ```suggestion
<!-- As a result, we may not immediately (or ever) get around to reviewing pull requests that do not support the Framework -->
```
